DumbCharades/ContentView.swift

```diff
@@ -55,18 +59,17 @@ struct ContentView: View {
                 }
                 .contentShape(Rectangle())
                 .background(Color.orange)
-                
+            
+            HStack {
+                Button(action: {
+                    isTimerActive = !isTimerActive
+                }) {
+                    Image(systemName: isTimerActive ? pauseIcon : playIcon)
+                }
+                .foregroundColor(Color.white)
+
                 Text("Remaining - \(timeRemaining)")
-                    .font(.system(size: 20, weight: .medium, design: .default))
-                    .foregroundColor(Color.orange)
-                    .padding(.horizontal, 20)
-                    .padding(.vertical, 5)
-                    .frame(maxWidth: .infinity)
-                    .background(
-                        Capsule()
-                            .fill(Color.black)
-                            .opacity(0.75)
-                    )
+                    .frame(width: 280)
```

Comment:
Does having fixed with cause display issues for different sizes?

Reply:
should be good now